### PR TITLE
feat(main): added github action to build and push OGBV ML rest server

### PR DESCRIPTION
**Pull request by Bhargav-Dave**

### CODE REVIEW

File: .github/workflows/deploy-staging.yml [hunk 1]
```diff
@@ -61,3 +64,33 @@ jobs:
...
+          ref: development
+      - 
+        name: Declare some variables
+        id: vars
+        shell: bash
+        run: |
+          echo "setting variables"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+          
+      -
```
Comment (dennyabrain) [inline .github/workflows/deploy-staging.yml:85]: @Bhargav-Dave looks like this indentation/formatting issue is there on all steps of your job. can you fix and push?

**Comment by dennyabrain** [comment]
remember to mention the issue this is addressing in a comment here. I'll add that to the commit before merging.

File: .github/workflows/deploy-staging.yml (hunk 1 above)
Comment (Bhargav-Dave) [reply .github/workflows/deploy-staging.yml:85]: Sure, doing that

**Comment by Bhargav-Dave** [comment]
It's issue #128.

**Pull request closed without merging**